Gulpfile.js

```diff
@@ -1175,3 +1175,13 @@ export function downloadSidecar() {
 
   return result;
 }
+
+prepareHusky.description = "Initialize Husky Git hooks for pre-commit linting.";
+export function prepareHusky(done) {
+  const result = spawnSync("npx", ["husky"], {
+    stdio: "inherit",
+    shell: IS_WINDOWS,
```

Comment:
Windows requires shell execution for npx to work correctly, this runs it conditionally in the shell